test/style.scss

```diff
@@ -1,55 +1,57 @@
 @import "../src/fluidms";
 
+$h1-font-size: ms(6) !default;
```

Comment:
Why is there a module variable for h1 and all subsequent headings (2-6) have a "fixed" value applied?

Reply:
This is really just a demo usage but fair enough, I updated it.